acceptancetests/assess_caas_deploy_kubeflow.py

```diff
@@ -410,8 +411,12 @@ def success_hook():
             caas_client.kubectl('get', 'sa,roles,clusterroles,rolebindings,clusterrolebindings', '-oyaml', '-A'),
         )
 
+    def success_hook():
+        dump_success_logs()
+        caas_client.ensure_cleanup()
```

Comment:
Could we have a comment here to explain why we need to delete the namespace before we have proper teardown (probably just copy the comment below for `bs_manager.keep_env = False`)?

Reply:
Done.